packet/models.py

```diff
@@ -95,19 +95,23 @@ def signatures_received(self):
 
         return SigCounts(upper, fresh, len(self.misc_signatures))
 
-    def did_sign(self, username, is_csh):
+    def did_sign(self, username, is_csh, is_frosh):
         """
         :param username: The CSH or RIT username to check for
         :param is_csh: Set to True for CSH accounts and False for freshmen
```

Comment:
Add a description of is_frosh in the docs

Reply:
There's the docstring in the function https://github.com/ComputerScienceHouse/packet/pull/245/files#diff-676fa9ab92e4191b3021cd107306160a00fb4ac435e08f94f7be212019d2b846R269-R278

Did you mean somewhere else?

Reply:
Oh, in the did_sign docstring